RELEASE_NOTES.md

```diff
@@ -1,5 +1,23 @@
 # Release Notes
 
+## stable-1.18-3.4.0, stable-1.19-3.4.0, v3.4.0
+
+### opsportal
+  - Fixes bug in OpsPortal where Identity Providers would not show up
+  #954 (@mesosphere-mergebot)
```

Comment:
I am unsure how it should be formatted in these release notes, but this fix is for a COPS (i.e. customer issue) and so we need to include the COPS number here:

```suggestion
  #954 (@mesosphere-mergebot) (COPS-6843)
```

Reply:
Please edit the description of #954 and it's parent and sibling PRs and add that to the release notes block.